### PR TITLE
Featuring Memecoin

### PR DESCRIPTION
**Pull request by bgodlin**

### CODE REVIEW

**Review by deverka — APPROVED** [review]

File: Fantom/fantom-memecoins/README.md [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+3.  `docker-compose pull && docker-compose up` - Runs a Docker container with an indexer, PostgeSQL DB, and a query service. This requires [Docker to be installed](https://docs.docker.com/engine/install) and running locally. The configuration for this container is set from your `docker-compose.yml`
+
+You can observe the three services start, and once all are running (it may take a few minutes on your first start), please open your browser and head to [http://localhost:3000](http://localhost:3000) - you should see a GraphQL playground showing with the schemas ready to query. [Read the docs for more information](https://academy.subquery.network/run_publish/run.html) or [explore the possible service configuration for running SubQuery](https://academy.subquery.network/run_publish/references.html).
+
+## Query your project
+
+For this project, you can try to query with the following GraphQL code to get a taste of how it works.
+
+```graphql
+{
```
Comment (jamesbayly) [inline Fantom/fantom-memecoins/README.md:42]: Update this to include an example balances query

File: Fantom/fantom-memecoins/src/mappings/mappingHandlers.ts [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+const erc20 = Erc20Abi__factory.connect(contractAddress, api);
+
+async function getOrCreateAddress(accountAddress: string): Promise<Address> {
+  let address = await Address.get(accountAddress);
+
+  if (address == undefined) {
+    address = Address.create({
+      id: accountAddress,
+      balance: (await erc20.balanceOf(accountAddress)).toBigInt(),
```
Comment (jamesbayly) [inline Fantom/fantom-memecoins/src/mappings/mappingHandlers.ts:18]: Should you also update the balance of an address if it's known (e.g. refresh this on each new transaction?)

Comment (jamesbayly) [reply Fantom/fantom-memecoins/src/mappings/mappingHandlers.ts:18]: Ahh I see you do this below

File: Fantom/fantom-memecoins/src/mappings/mappingHandlers.ts [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    id: log.transactionHash + "-" + log.logIndex,
+    blockHeight: BigInt(log.blockNumber),
+    toId: to.id,
+    fromId: from.id,
+    value: value,
+    contractAddress: log.address,
+  });
+
+  from.balance = BigInt(from.balance) - value;
+  to.balance = BigInt(to.balance) + value;
```
Comment (jamesbayly) [inline Fantom/fantom-memecoins/src/mappings/mappingHandlers.ts:44]: I think it's safer to do a full `await erc20.balanceOf(accountAddress)` There might be other functions and activities that change this so best to clean reset.

Use Promise.all([..,..] for performance for these two queries

**Review by jamesbayly — COMMENTED** [review]
Mostly there, just a few more things

**Pull request merged**